safe/safe.go

```diff
@@ -72,14 +71,15 @@ func (s *Safe) IsVersionCompatible(version string) error {
 		return nil
 	}
 
-	incompatible, err := versionIncompatible(s.Config.Version, version) // nolint: vetshadow
+	incompatible, err := versionIncompatible(s.Config.Version, version)
 	if err != nil {
 		return err
-	} else if incompatible {
+	}
+	if incompatible {
```

Comment:
The `else if` was fine here. Whatever.

Reply:
I'm not a huge fan of `else if` in these cases, personal preference 😬